README.md

```diff
@@ -1,107 +1,115 @@
 
-### Build the Environment
+## Build and Install Gluten with Arrow backend
 
-Gluten can only support single Spark version now. In Gluten main branch, Spark3.2.x is the only version to be supported currently. If you would like to try TPCH queries on Spark3.1.x, you may find [Gluten with Spark3.1.1](https://github.com/oap-project/gluten/tree/spark311) to give a try, but it is no longer to be supported or maintained.
-We are working on a shim layer to support multiple Spark version in the future.
+If you would like to build and try Gluten with **Arrow** backend, please follow the steps in [Build with Arrow Backend](./docs/ArrowBackend.md). Arrow backend only support parquet scan and parquet write now. All other operators are fallback to Vanilla Spark.
 
-You can build the environment by yourself.
+## Build script parameters
 
-### Build Gluten with Velox backend
+[Gluten Usage](./docs/GlutenUsage.md) listed the parameters and their default value of build command for your reference
 
-After Gluten being successfully deployed in your environment, if you would like to build Gluten with **Velox** computing, please follow the steps in [Build with Velox](./docs/Velox.md) to install the needed libraries, compile Velox and try out the TPC-H Q6 and Q1 test.
+# Contribution
 
-### Build Gluten with Arrow backend
+Gluten project welcomes everyone to contribute. 
 
-If you would like to build Gluten with **Arrow** backend, please follow the steps in [Build with Arrow Backend](./docs/ArrowBackend.md).
+## Community
 
-### Build Gluten with ClickHouse backend
+Currently we communite with all developers and users in a wechat group(Chinese only), Spark channel in Velox Slack group. Contact us if you would like to join in.
 
-![logo](./docs/image/ClickHouse/logo.png)
+## Bug Reports
 
-If you would like to build Gluten with **ClickHouse** backend, please follow the steps in [Build with ClickHouse Backend](./docs/ClickHouse.md). ClickHouse backend is devleoped by [Kyligence](https://kyligence.io/), please visit https://github.com/Kyligence/ClickHouse for more infomation.
+Feel free to submit any bugs, issues or enhancement requirements to github issue list. Be sure to follow the bug fill template so we can solve it quickly. If you already implement a PR and would like to contribute, you may submit an issue firstly and refer to the issue in the PR. 
 
-### Build script parameters
+## Documentation
 
-[Gluten Usage](./docs/GlutenUsage.md) listed the parameters and their default value of build command for your reference
+Unfortunately we haven't create the orginazed documentation site for Gluten. Currently all document is hold in [docs](https://github.com/oap-project/gluten/tree/main/docs). Ping us if you would like to know more details about the Gluten design. Gluten is still in developping now, some designs may change. Feel free to talk with us if you have better design.
```

Comment:
Gluten is still under development now, and some designs may change. Feel free to talk with us and share other design and ideas.